build.sbt

```diff
@@ -314,7 +321,7 @@ lazy val `service-integration-tests` = (project in file("service-integration-tes
     PgpKeys.publishSigned := {},
     publish := {}
   )
-  .dependsOn(server, persistence, pubsub, testkit, logback)
+  .dependsOn(server, persistence, pubsub, testkit, logback, `integration-client`)
```

Comment:
First reaction is that I don't see why the `server` project should have a dependency on the `integration-client`. I'd rather have users add an explicit dependency to `integration-client` when they need it.